setup.py

```diff
@@ -0,0 +1,3 @@
+from setuptools import setup, find_packages
```

Comment:
c'est censé installer l'environnement du projet direct tout seul ? genre toutes les imports etc ??

Reply:
fixed, will be handled in future